### PR TITLE
Plans docs

### PR DESCRIPTION
**Pull request by hugovalente-pm**
##### Summary

Initial PR for the pages on:
* Netdata Plans
* View Plan & Billing

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

**Comment by hugovalente-pm** [comment]
this is an initial PR, these docs can and should be improved with concrete examples of P90 calculations, different plan subscriptions and overage situations on yearly ones.
didn't want to have a blocker on those since these can be added alter

File: docs/cloud/manage/plans.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+It is ok to change your mind. We allow to change your plan, billing frequency or adjust the committed nodes, on yearly plans, at any time.
+
+To achieve this you will need to:
+* Move to the Community plan, where we will cancel the current subscription and:
+   * Issue a credit to you for the unused period, in case you are on a **yearly plan**
+   * Charge you only for the current used period and issue a credit for the unused period related to the flat fee, in case you are on a **monthly plan**
+* Select the new subscription with the change that you want
+
+> ⚠️
```
Comment (cakrit) [inline docs/cloud/manage/plans.md:60]: Make these two separate warnings, doesn't look good this way.

File: docs/cloud/manage/plans.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+> * On a move to Community, cancellation of an active subscription, please note that you will have all your notification methods configurations active for a period of 24 hours.
```
Comment (cakrit) [inline docs/cloud/manage/plans.md:61]:
```suggestion
> * On a move to Community (cancellation of an active subscription), please note that you will have all your notification methods configurations active for a period of 24 hours.
```

File: docs/cloud/manage/plans.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+>   After that, any of those that won't available on your space's plan at that time will be automatically disabled. You can always re-enable them once you move to the proper paid plan.
```
Comment (cakrit) [inline docs/cloud/manage/plans.md:62]:
```suggestion
>   After that, any notification methods unavailable in your new plan at that time will be automatically disabled. You can always re-enable them once you move to a paid plan that includes them.
```

**Review by cakrit — CHANGES_REQUESTED** [review]
Very minor corrections here.

**Review by cakrit — APPROVED** [review]

**Review by cakrit — CHANGES_REQUESTED** [review]
This PR suddenly includes a bunch of changes to the netdata code. Something happened between master and your branch Hugo, please rebase or start again with a new branch and PR.

**Comment by hugovalente-pm** [comment]
> This PR suddenly includes a bunch of changes to the netdata code. Something happened between master and your branch Hugo, please rebase or start again with a new branch and PR.

@cakrit did a fix that Tasos told me about, this sometimes happens because some issue with GitHub. all should be good now

**Review by cakrit — APPROVED** [review]

**Pull request merged**